gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/test/java/io/gravitee/am/gateway/handler/common/auth/UserAuthenticationManagerTest.java

```diff
@@ -42,9 +43,7 @@
 import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
 
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.LinkedHashSet;
+import java.util.*;
```

Comment:
I think we should try to avoid `*` imports, even in tests (and especially with `java.util.*`).

Reply:
The main drawback of using wildcard imports is possible naming conflicts / readability.
I think this is something that needs to be set up in the future and also in our IDEs